### PR TITLE
Minecraft: British spelling of colour

### PR DESCRIPTION
**Pull request by kirkins**
## Description of new Instant Answer, or changes

This matches British spelling of colour for minecraft guide.

Will re-base with #4271 if it's finished and pulled before this one.

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/minecraft

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/Minecraft.t [hunk 1]
```diff
@@ -140,7 +165,7 @@ ddg_goodie_test(
 sub make_structured_answer {
     my ($name, $ingredients, $description, $image ) = @_;
     my %recipe = (
-	    name => $name,
+	name => $name,
```
Comment (mintsoft) [inline t/Minecraft.t]: Can you fix the hard-tab here? it should be 4 spaces

Comment (kirkins) [inline t/Minecraft.t]: @mintsoft got it 👍

Comment (mintsoft) [inline t/Minecraft.t]: @kirkins, there's still a tab here ?

Comment (kirkins) [inline t/Minecraft.t]: @mintsoft no not really. I can see why you'd think that from the above but try highlighting the white space on this [page](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/4272/commits/13cb5a40b60692cf49dc5b0518920a4da302bbc3).

**Pull request merged**